.travis.yml

```diff
@@ -44,6 +46,12 @@ install:
     - cd ${SRC_DIR}
     - pip install -e .
 
+before_script:
+    # See https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI :
+    - "export DISPLAY=:99.0"
+    - "sh -e /etc/init.d/xvfb start"
+    - sleep 3 # give xvfb some time to start
```

Comment:
so this is also not needed ...

Reply:
You can try to remove it but you had an issue related to xvfb with your first PR if I remember correctly